### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by ocornoc**
For Travis-CI integration

### CODE REVIEW

File: tests/composition/multi_type.cpp [hunk 1]
```diff
@@ -24,33 +22,44 @@ secondf_ret_t func2(secondf_arg_t n)
 	return n > 10000;
```
Comment (osilkin98) [inline tests/composition/multi_type.cpp]: this has nothing to do with the fixes but that loose 10000 value needs to be assigned to some actual constant

**Comment by ocornoc** [comment]
Good point, I'll make a commit later (or you can) to assign it to a constexpr static variable

File: tests/composition/multi_type.cpp (hunk 1 above)
Comment (ocornoc) [reply tests/composition/multi_type.cpp]: Good point, I'll make a commit later (or you can) to assign it to a constexpr static variable

File: tests/composition/multi_type.cpp [hunk 2]
```diff
@@ -24,33 +22,44 @@ secondf_ret_t func2(secondf_arg_t n)
...
+	volatile secondf_ret_t ret = comp(n).get();
 	
-	std::cout << std::endl;
+	return ret;
 }
 
 int main()
 {
-	firstf_arg_t n = 0;
+	for (int i = 9990; i <= 10000; i++) {
```
Comment (osilkin98) [inline tests/composition/multi_type.cpp]: Consider either removing the main function body or giving some small documentation on the examples

File: tests/composition/single_type.cpp [hunk 3]
```diff
@@ -20,42 +19,22 @@ ret_t func3(ret_t n)
 	return (n - 1) / 2;
 }
 
-void test_composition(ret_t n)
+bool test_composition(ret_t n)
```
Comment (osilkin98) [inline tests/composition/single_type.cpp]: You should add a `#define composition composition<ret_t(ret_t)>` directive so it gets rid of the STL brackets that make it look atrocious. Consider doing this for all template functions that you have so your code looks cleaner.

Comment (ocornoc) [reply tests/composition/single_type.cpp]: I'll use a `using` alias/`typedef` instead

**Pull request closed without merging**